logagg/collect/collector.py

```diff
@@ -155,7 +164,17 @@ def send_to_nsq(self):
 
             try:
                 if should_push:
+                    counter = 0
+                    self.SLEEP_TIME = 1
                     while 1:
+                        if counter < 5 and counter > 0:
+                            self.SLEEP_TIME += 1
```

Comment:
@praneeth36 exponential back off would mean the sleep time would be multiplied by 2, not added by one.